tests/testthat/examples/Test_2023_2/output/taxa.csv

```diff
@@ -1,46 +1,46 @@
-taxon_name,taxonomic_reference,taxon_rank,taxonomic_status,genus,binomial,trinomial,family,taxon_id,taxon_distribution
-Acacia celsa,APC,species,accepted,Acacia,Acacia celsa,,Fabaceae,https://id.biodiversity.org.au/taxon/apni/51436506,Qld
-Acmena graveolens,,,,,,,,,
-Acronychia acidula,APC,species,accepted,Acronychia,Acronychia acidula,,Rutaceae,https://id.biodiversity.org.au/node/apni/2913200,Qld
-Aleurites rockinghamensis,,,,,,,,,
-Alphitonia petriei,APC,species,accepted,Alphitonia,Alphitonia petriei,,Rhamnaceae,https://id.biodiversity.org.au/node/apni/2887911,"Qld, NSW"
-Alstonia scholaris,,,,,,,,,
-Argyrodendron peralatum,,,,,,,,,
-Atractocarpus hirtus,,,,,,,,,
-Brombya platynema,,,,,,,,,
-Cardwellia sublimis,,,,,,,,,
-Castanospermum australe,,,,,,,,,
-Cleistanthus myrianthus,,,,,,,,,
-Cryptocarya laevigata,,,,,,,,,
-Cryptocarya mackinnoniana,,,,,,,,,
-Cryptocarya murrayi,,,,,,,,,
-Dendrocnide moroides,,,,,,,,,
-Elaeocarpus grandis,,,,,,,,,
-Endiandra leptodendron,,,,,,,,,
-Endiandra microneura,,,,,,,,,
-Gillbeea adenopetala,,,,,,,,,
-Glochidion hylandii,APC,species,accepted,Glochidion,Glochidion hylandii,,Phyllanthaceae,https://id.biodiversity.org.au/node/apni/2889113,Qld
-Haplostichanthus submontanus,,,,,,,,,
-Harpullia rhyticarpa,,,,,,,,,
-Hernandia albiflora,,,,,,,,,
-Homalanthus novo-guineensis,,,,,,,,,
-Ixora biflora,,,,,,,,,
-Lasianthus strigosus,,,,,,,,,
-Litsea leefeana,,,,,,,,,
-Macaranga tanarius,,,,,,,,,
-Mallotus mollissimus,,,,,,,,,
-Medicosma sessiliflora,,,,,,,,,
-Melastoma cyanoides,,,,,,,,,
-Melicope elleryana,APC,species,accepted,Melicope,Melicope elleryana,,Rutaceae,https://id.biodiversity.org.au/node/apni/2913641,"WA, NT, Qld, NSW"
-Myristica insipida,,,,,,,,,
-Neolitsea dealbata,,,,,,,,,
-Pittosporum rubiginosum,,,,,,,,,
-Polyscias australiana,,,,,,,,,
-Psychotria dallachiana,,,,,,,,,
-Psychotria sp. Utchee Creek (H.Flecker NQNC5313),,,,,,,,,
-Quassia baileyana,,,,,,,,,
-Rhodomyrtus trineura,,,,,,,,,
-Rockinghamia angustifolia,,,,,,,,,
-Syzygium gustavioides,,,,,,,,,
-Syzygium sayeri,,,,,,,,,
-Trema aspera,,,,,,,,,
+aligned_name,taxon_name,taxonomic_dataset,taxon_rank,taxonomic_status,genus,binomial,trinomial,family,taxon_id,taxon_distribution
+Acacia celsa,Acacia celsa,APC,species,accepted,Acacia,Acacia celsa,,Fabaceae,https://id.biodiversity.org.au/taxon/apni/51436506,Qld
+,Acmena graveolens,,,,,,,,,
```

Comment:
did you mean to add a blank cell at the start of each row?

Reply:
removed them again - had added `aligned_name`